### PR TITLE
Quantcast: Adding custom label support

### PR DESCRIPTION
**Pull request by harrietgrace**
Support for adding extra labels in the properties object, and sending a comma separated list of labels to Quantcast.

Quantcast docs: https://www.quantcast.com/help/labeling-and-hierarchy

### CODE REVIEW

**Comment by harrietgrace** [comment]
@ndhoule @amillet89 All good to go for review!

File: lib/quantcast/index.js [hunk 1]
```diff
@@ -175,7 +182,7 @@ Quantcast.prototype.labels = function(type){
   for (var i = 0; i < args.length; ++i) {
     if (null == args[i]) continue;
     var value = String(args[i]);
-    ret.push(value.replace(/,/g, ';'));
+    ret.push(value.replace(/, /g, ','));
   }
```
Comment (ndhoule) [inline lib/quantcast/index.js]:
```javascript
var separator = advertise ? ' ' : '.';
var ret = reduce(args, function(ret, arg) {
  if (arg != null) {
    ret.push(String(arg).replace(/, /g, ','));
  }
  return ret;
}, []).join(separator);
```

File: lib/quantcast/index.js [hunk 2]
```diff
@@ -68,9 +68,11 @@ Quantcast.prototype.loaded = function(){
 Quantcast.prototype.page = function(page){
   var category = page.category();
   var name = page.name();
+  var customLabels = page.proxy('properties.label')
+
   var settings = {
     event: 'refresh',
-    labels: this.labels('page', category, name),
+    labels: this.labels('page', category, name, customLabels),
```
Comment (ndhoule) [inline lib/quantcast/index.js]: Assign this to a variable for consistency with below functions

File: lib/quantcast/index.js (hunk 1 above)
Comment (harrietgrace) [inline lib/quantcast/index.js]: That's even cooler. Chain all the things.

**Comment by ndhoule** [comment]
lgtm

**Pull request merged**